ssg/entities/profile_base.py

```diff
@@ -55,6 +55,10 @@ class Profile(XCCDFEntity, SelectionHandler):
         "selections",
```

Comment:
@jan-cerny Another approach would be to remove `selections` from `MANDATORY_KEYS`,  I don't think this would open gaps that could lead to problems. The `ALTERNATIVE_KEYS` seem guard and lead better to what is expected.